internal/cri/util/util.go

```diff
@@ -30,7 +30,7 @@ import (
 
 // deferCleanupTimeout is the default timeout for containerd cleanup operations
 // in defer.
-const deferCleanupTimeout = 1 * time.Minute
+const deferCleanupTimeout = 5 * time.Minute
```

Comment:
I wonder if we should use the `time`(https://github.com/containerd/containerd/blob/main/pkg/timeout/timeout.go) package for this and expose it in the config (see examples [here](https://github.com/containerd/containerd/blob/main/core/runtime/v2/shim.go#L57-L67) and [here](https://github.com/containerd/containerd/blob/main/core/runtime/v2/shim_load.go#L205))